mistralrs-server/src/main.rs

```diff
@@ -112,6 +120,13 @@ impl futures::Stream for Streamer {
                 Response::ValidationError(e) => {
                     Poll::Ready(Some(Ok(Event::default().data(e.to_string()))))
                 }
+                Response::ModelError(msg, _) => {
+                    MistralRs::maybe_log_error(
+                        self.state.clone(),
+                        &ModelErrorMessage(msg.to_string()),
+                    );
+                    Poll::Ready(Some(Ok(Event::default().data(msg))))
+                }
```

Comment:
`candle_core::Error` does not implement clone or copy, so I could not duplicate it to set as the response for every sequence...

So I decided to carry it around as a string. That's unfortunate here, where an ad-hoc error needs to be created

Reply:
I think it should be fine: we only need to optimize the happy path, as the hope is that there will be no errors!